include/revng/Pypeline/Helpers/Python/RunnerInfo.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "nanobind/nanobind.h"
+
+namespace revng::pypeline::helpers::python {
+
+// Helper class to unpack containers from a nanobind::list
+// To be used in conjunction with PipeRunner or AnalysisRunner
+template<typename T>
+class RunnerInfo {
```

Comment:
1. Bad name. What about `ContainerListUnwrapper`?
2. I'd move this to `Utils.h`

Reply:
1. Renamed to that
2. Done